_targets.R

```diff
@@ -253,6 +259,19 @@ list(
              format = "file"
   ),
   
+  ##generate table of data to download from sciencebase
+  tar_target(p1_make_sb_dl_table,
+             makedltable(idtostart, outdir = "./1_fetch/out"),
+             format = "file"),
+  
+  ##generate table of landscape data for gages list  
+  tar_target(p1_make_sb_landscapedata,
+             downloadchildren(gages=gagesii, dldir = "./1_fetch/out/dldir", 
+                              workdir = "./1_fetch/out/workdir",
+                              outdir = "./1_fetch/out",
+                              table_sb_dl = p1_make_sb_dl_table),
+             format = "file"),
```

Comment:
@jds485 The target 'p1_make_sb_dl_table' is pretty quick, so I'm guessing that would be deployment = 'main'. The target 'p1_make_sb_landscapedata' is slower, so I'm tempted to assign deployment = 'worker' but I suspect it will not be that easy. Would like to discuss this with you.